cli/flox/src/commands/init/node.rs

```diff
@@ -311,19 +479,15 @@ impl Node {
         Ok((message, version, mentions_package_json))
     }
 
-    fn prompt_with_known_hook(&self, flox: &Flox) -> Result<bool> {
+    /// Prompt whether to install nodejs (but not npm or yarn)
+    fn prompt_for_node(&self, flox: &Flox) -> Result<bool> {
         let (nodejs_detected, nodejs_version, mentions_package_json) =
             self.nodejs_message_and_version(flox)?;
         let mut message = format!("{nodejs_detected}\n");
-        match self.hook {
-            None => {},
-            Some(NodePackageManager::Npm) if !mentions_package_json => {
-                message.push_str("Flox detected a package.json\n")
-            },
-            Some(NodePackageManager::Npm) => {},
-            Some(NodePackageManager::Yarn) => message.push_str("Flox detected a yarn.lock\n"),
-            Some(NodePackageManager::Both) => unreachable!(),
+        if !mentions_package_json {
+            message.push_str("Flox detected a package.json\n");
         }
+
         message.push_str(&formatdoc! {"
 
             Flox can add the following to your environment:
```

Comment:
shouldn't this say "nodejs and an npm hook"?

Reply:
Good catch, pushed a change to conditionally add that when `node_install.npm_hook` is true